src/fragmentarium/domain/pager.ts

```diff
@@ -0,0 +1,12 @@
+export interface NextAndPrevFragment {
+  readonly previous: string
+  readonly next: string
+}
+type FragmentAndFolio = {
```

Comment:
This should also be an `interface` for consistency.

In fact, `interface` should be favoured over `type`: [Interfaces vs. Type Aliases](https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases)